### PR TITLE
mention uppy_s3_multipart in doc/direct_s3

### PR DESCRIPTION
**Pull request by jrochkind**

### CODE REVIEW

File: doc/direct_s3.md [hunk 1]
```diff
@@ -177,6 +177,10 @@ upload walkthrough] for adding dynamic direct S3 uploads from scratch, as well
 as the [Roda][roda demo] / [Rails][rails demo] demo app for a complete example
 of multiple direct S3 uploads.
 
+Also, if you're dealing with larger files, you may want to make the uploads resumable
+by using the AwsS3Multipart Uppy plugin instead, with the [uppy-s3_multipart](https://github.com/janko-m/uppy-s3_multipart) gem on the backend. Your back-end implementation is similar, just using
+`Shrine.presign_endpoint` in place of `Shrine.uppy_s3_multipart`. Instructions can be found in uppy-s3-multipart README.
```
Comment (janko-m) [inline doc/direct_s3.md]: Was it supposed to be the other way around, "just using `Shrine.uppy_s3_multipart` in place of `Shrine.presign_endpoint`"?

Comment (janko-m) [reply doc/direct_s3.md]: Also, small thing, but instead of "uppy-s3-multipart" it should be "uppy-s3_multipart" here.

Comment (jrochkind) [reply doc/direct_s3.md]: You're right on order, will fix. 

The `mount` command is `mount Shrine.uppy_s3_multipart`, with all underscores. So I think that's correct there, as telling you the mounting methods involved that you'll be swapping. Earlier when I mention and link to the them, it is correctly "uppy-s3_multipart"

Comment (jrochkind) [reply doc/direct_s3.md]: oops, I see the place I had wrong too many underscores, sorry you're right, will fix!

**Comment by jrochkind** [comment]
I was trying to figure out why the travis tests were failing -- becuase you merged and deleted the branch before travis started all jobs, so travis couldn't pull the (deleted) branch to test it. OK! Thanks!

**Comment by janko-m** [comment]
Oh, oops, I will let it finish next time or cancel it.

**Pull request merged**